mu-plugins/blocks/notice/postcss/style.pcss

```diff
@@ -0,0 +1,78 @@
+.wp-block-wporg-notice {
+	--wp--custom--wporg-notice--color--background: var(--wp--preset--color--acid-green-3);
+	--wp--custom--wporg-notice--color--text: var(--wp--preset--color--charcoal-1);
```

Comment:
When I look at other notice-like components, the links are still the default link color:

<img width="307" alt="Screen Shot 2023-01-19 at 9 28 54 AM" src="https://user-images.githubusercontent.com/1657336/213327471-a6203793-5781-4027-9cd4-0f84251628bf.png">

Should we also use that color? Or maybe make the component support link color updates?